src/app.js

```diff
@@ -1 +1,42 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { numGenerator } = require('./numGenerator.js');
+const readline = require('readline');
+
+const digitsToGuess = numGenerator();
+
+const termianl = readline.createInterface({
```

Comment:
This typo really needs to be corrected; you really shouldn't let it slide

Reply:
+1